templates/sparkjob.json

```diff
@@ -2,28 +2,40 @@
    "kind": "Template",
    "apiVersion": "v1",
    "metadata": {
-      "name": "oshinko-pyspark-job",
+      "name": "oshinko-spark-job",
       "annotations": {
-         "description": "Create a job using an existing pyspark imagestream"
+         "description": "Create a job using an existing Spark image"
       }
    },
    "labels": {
-      "application": "oshinko-pyspark",
-      "createdBy": "template-oshinko-pyspark-job"
+      "application": "oshinko-spark",
+      "createdBy": "template-oshinko-spark-job"
    },
    "parameters": [
       {
          "description": "The name to use for the job",
          "name": "APPLICATION_NAME",
          "generate": "expression",
-         "from": "pyspark-[a-z0-9]{4}",
+         "from": "spark-[a-z0-9]{4}",
          "required": true
       },
       {
-         "description": "The docker pull specification of the image to run",
+         "description": "The Docker pull specification of the image to run",
```

Comment:
nit: we should slowly start doing `s/Docker/container/g` everywhere :)

Reply:
Good idea.

I am going to add some tests, and also eliminate "app" labels in another PR, maybe I can do that too.

Reply:
i think we should follow the upstream openshift docs, originally these values were also referred to as the docker pull spec.